crates/ide-assists/src/tests.rs

```diff
@@ -1000,10 +1026,12 @@ pub fn test_some_range(a: int) -> bool {
                                         Indel {
                                             insert: "fun_name()",
                                             delete: 59..60,
+                                            annotation: None,
                                         },
                                         Indel {
                                             insert: "\n\nfn fun_name() -> i32 {\n    5\n}",
                                             delete: 110..110,
+                                            annotation: None,
```

Comment:
nit but might be nice to omit rendering annotation in the debug impl if its `None` (given we use it in a bunch of expect tests that reduces some noise)

Reply:
I prefer to update the tests, like what if we will need it in some of them eventually.